invokeai/app/services/config/config_default.py

```diff
@@ -147,7 +145,7 @@ class InvokeAIAppConfig(BaseSettings):
 
     # PATHS
     models_dir:                    Path = Field(default=Path("models"),     description="Path to the models directory.")
-    convert_cache_dir:             Path = Field(default=Path("models/.convert_cache"), description="Path to the converted models cache directory. When loading a non-diffusers model, it will be converted and store on disk at this location.")
+    convert_cache_dir:             Path = Field(default=Path("models/.convert_cache"), description="Path to the converted models cache directory (DEPRECATED).")
```

Comment:
Done. I’ve gone ahead and removed `convert_cache`, however, because it doesn’t hurt anything and does illustrate one approach to migrating the config schema.